lib/Factory/TagFactory.php

```diff
@@ -383,10 +383,9 @@ public function query($sortOrder = null, $filterBy = [])
         }
 
         //isSystem filter, by default hide tags with isSystem flag
-        if ($sanitizedFilter->getCheckbox('isSystem') === 1) {
-            $body .= " AND `tag`.isSystem = 1 ";
-        } else {
-            $body .= " AND `tag`.isSystem = 0 ";
+        if ($sanitizedFilter->getCheckbox('isSystem') !== null && $sanitizedFilter->getInt('allTags') !== 1) {
```

Comment:
I don't think `getCheckbox` can return null - only true/false?

EDIT: what I mean is, won't this statement always be true because true !== null and false is also !== null :-)